packages/protocol/contracts/governance/test/ProposalsTest.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.8;
+
+import "../Proposals.sol";
+
+
+contract ProposalsTest {
```

Comment:
Did you consider testing other Proposal functions?

Reply:
The primary functions (`make`, `vote`, `execute`) used in external Governance functions are already logically tested in the Governance tests, which I opted not to mock and decouple because the library as a whole is expected to be localized to Governance (also a lot of work). All other functions, apart from the one tested, seemed trivial enough to leave alone.